apps/ledger-live-mobile/src/screens/Settings/General/WalletSyncRow.tsx

```diff
@@ -27,21 +33,27 @@ const WalletSyncRow = () => {
         screen: ScreenName.WalletSyncActivated,
       });
     } else {
-      navigation.navigate(NavigatorName.WalletSync, {
-        screen: ScreenName.WalletSyncActivationSettings,
-      });
```

Comment:
We can remove it from screen so?

Reply:
Only the activation Flow is inside a drawer. This screen is mandatory for when the user has already a trustchain